kube-scheduler-healthcheck

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -eu
+TEST_POD_NAMESPACE="${TEST_POD_NAMESPACE:-kube-system}"
+TEST_POD_IMAGE="${TEST_POD_IMAGE:-gcr.io/google_containers/pause:3.0.0}"
+
+cat > /tmp/pod.yaml <<endOfPodDef
+apiVersion: v1
+kind: Pod
+metadata:
+  name: scheduler-pod-healthcheck
+spec:
+  containers:
+  - name: pause
+    image: ${TEST_POD_IMAGE}
+endOfPodDef
+
+alias kubectl_and_args='kubectl -n $TEST_POD_NAMESPACE --request-timeout=30s'
```

Comment:
sorry, this actually needs to be:
```
alias kubectl="kubectl --namespace \$TEST_POD_NAMESPACE --request-timeout=30s"
```
single quotes won't work and need to escape the dollar

Please use long form parameters in scripts to make as clear as possible when reading

Reply:
and just make the alias `k` or `kubectl`, no need to add other stuff to it

Reply:
why? to me `kubectl_and_args` is clearer than just `kubectl` or `k`. could miss that line and assume it's default `kubectl`.